perfkitbenchmarker/aws/aws_network.py

```diff
@@ -151,21 +151,27 @@ def _Delete(self):
 class AwsSubnet(resource.BaseResource):
   """An object representing an Aws subnet."""
 
-  def __init__(self, zone, vpc_id):
+  def __init__(self, zone, vpc_id, cider_block=None):
```

Comment:
Also please set the default value in the constructor:

```{python}
def __init__(self, zone, vpc_id, cidr_block='10.0.0.0/24'):
```

It's more visible that way, and simplifies setting the member variable.